### ISSUE TITLE
Add option to allow or prohibit html attributes

### ISSUE THREAD

**Issue by FrancisVila** [DavidAnson/markdownlint#464]
I'd like to set the markdownlint.json file so that it allows HTML tags such as `<table>`, `<tr>`, `<td>`... tags, but prohibits any attributes like `onclick `or `onmouseover`. I've found the `"MD033": { "allowed_elements": ["table", "tr"]}` entry, but no way of banning or allowing specific attributes within those tags. 

Something like 
```
"MD033": { 
   "allowed_elements": ["table", "tr"], 
   "allowed_attributes": ["class", "style"]
}
```

Why: I want to avoid the risk of accidentally letting through malicious code inside Pull Requests. I'm also converting docs from XHTML generated by Madcap Flare into markdown. Some of the tables are complex, with tables inside tables, multiple uses of colspan and rowspan. I am setting aside such tables from the conversion process (I'm using pypandoc in python, a thin wrapper for pandoc).

**Comment by DavidAnson** [comment]
You **may** be able to block "onclick" today via MD044 by saying the preferred form is "OnClIcK": https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md044---proper-names-should-have-the-correct-capitalization

**Comment by FrancisVila** [comment]
Thanks, I'll try that!

**Comment by FrancisVila** [comment]
**Yes**, `'MD044': {'names': ['OnClIcK']},` does issue a warning for `<a href="#"  onclick="alert("HHHH")">AAA</a>`

**BUT** 

`<a href="#" OnClIcK="alert('toto')">EEEEEEEEEEE</a>` 
produces an alert, the same as  
`<a href="#"  onclick="alert('toto')">EEEEEEEEEEE</a>`

So someone knowing the trick could look up the `.markdownlint` file and check the specified capitalization...

**Comment by DavidAnson** [comment]
Someone who wants to bypass the rules can disable them with a comment. :) The linter is meant to help well-meaning people and is not a security boundary.

**Comment by FrancisVila** [comment]
Thanks David, I'll use that argument to justify adapting the .markdownlint file in my repo.
